e2e/appActions.js

```diff
@@ -129,7 +124,7 @@ async function openObjectTreeContextMenu(page, myItemsFolderName, domainObjectNa
 async function getFocusedObjectUuid(page) {
     const UUIDv4Regexp = /[0-9a-f]{8}-[0-9a-f]{4}-4[0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}/gi;
     const focusedObjectUuid = await page.evaluate((regexp) => {
-        return window.location.href.match(regexp).at(-1);
+        return window.location.href.split('?')[0].match(regexp).at(-1);
```

Comment:
should we use a [regular URL object](https://developer.mozilla.org/en-US/docs/Web/API/URL/searchParams) here instead?

Reply:
We should not! The query parameters in the OpenMCT URLs are not really query parameters, even though they look like them.

Reply:
As they're all put after the `#`